rules/linux/lnx_system_network_discovery.yml

```diff
@@ -0,0 +1,32 @@
+title: System Network Discovery - Linux
+id: e7bd1cfa-b446-4c88-8afb-403bcd79e3fa
+status: experimental
+description: Detects enumeration of local network configuration
+author: Ömer Günal and remotephone, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1016/T1016.md
+logsource:
+    category: process_creation
+    product: linux
+detection:
+    selection1:
+        Image|endswith:
```

Comment:
This causes a lot of FPs, as this is regularly used in administration, scripts etc.

Reply:
discussion here https://github.com/SigmaHQ/sigma/pull/1362#discussion_r598336458